apps/patient/src/components/StatusStepper/index.tsx

```diff
@@ -21,17 +23,25 @@ export const StatusStepper = ({ status, isCourier }: Props) => {
       <Container px={0} pt={0}>
         <Center>
           <Stack spacing="0">
-            {states.map((state, id) => (
-              <Step
-                key={state}
-                cursor="pointer"
-                title={t.status[fulfillmentType].states[state].title}
-                description={t.status[fulfillmentType].states[state].description}
-                isActive={currentStep === id}
-                isCompleted={currentStep > id}
-                isLastStep={states.length === id + 1}
-              />
-            ))}
+            {states.map((state, id) => {
+              const title = t.status[fulfillmentType].states[state].title
+              const description =
+                state === 'DELIVERING'
```

Comment:
Where does delivering come from?

Reply:
frontend

Reply:
I'm just matching the 4 pickup states to 4 courier states